rebar.config

```diff
@@ -25,11 +25,8 @@
                         , {eunit_surefire, [{dir, "."}]}
                         }]}.
 { deps
-, [ { jsx
-    , ".*"
-    , { git, "git://github.com/talentdeficit/jsx.git"
-      , {tag, "2.8.0"}
-      }
+, [ { jsx 
+    , "2.8.0"
```

Comment:
hex packages can only depend on hex packages not git dependencies. So it was required to move the jsx dependency from git to hex.